iris/static/js/lecturer_session.js

```diff
@@ -1,38 +1,42 @@
 const button_start = document.getElementById("button_start");
 const button_stop = document.getElementById("button_stop");
-const feedbackmsg = document.getElementById("lecturer_message")
+const feedbackmsg = document.getElementById("lecturer_message");
 let socket = io();
 socket.emit('join', {'course_id': courseID});
 
-if(sessionActive == "True"){
+function disableStart(){
     button_start.disabled = true;
     button_stop.disabled = false;
-}else{
+    feedbackmsg.innerHTML = "Session active";
+}
+
+function disableStop(){
     button_start.disabled = false;
     button_stop.disabled = true;
+    feedbackmsg.innerHTML = "Session not active";
+}
+
+if(sessionActive == "True"){
```

Comment:
1. Why is the variable "True", instead of a boolean true?
2. If this were a boolean, we could just use `if (sessionActive)`